security/deployment.adoc

```diff
@@ -128,34 +128,39 @@ There are two types of signature transports.
-API. Images must be stored by the OpenShift Container Registry. No additional
-configuration is required due to the *docker/distribution*
-xref:../admin_guide/image_signatures.adoc#reading-image-signatures-via-registry-api[`extensions` API] to auto-discover the image signature endpoint.
-
-Signatures using the `docker` transport type are served by local file or web
-server. They have the benefit of the most flexibility: any container registry
-can be used to serve images with an independent server to deliver binary
-signatures.
-
-However, the `docker` transport type requires additional configuration. Each
-node must be configured with the URI of the signature server using
-arbitrarily-named YAML files placed into a directory on the host system,
+The {product-title} API manages signatures that use the `atomic` transport type.
+You must store the images that use this signature type in the 
+the OpenShift Container Registry. Because the *docker/distribution* 
+xref:../admin_guide/image_signatures.adoc#reading-image-signatures-via-registry-api[`extensions` API]
+auto-discovers the image signature endpoint, no additional
+configuration is required.
+
+Signatures that use the `docker` transport type are served by local file or web
+server. These signatures are more flexible: you can serve images from any 
+container registry and use an independent server to deliver binary
+signatures. According to the
+link:https://github.com/containers/image/blob/master/docs/signature-protocols.md[Signature access protocols],
+you access the signatures for each image directly; the root of the server
+directory does not display its file structure.
+
+However, the `docker` transport type requires additional configuration. You must
+configure node with the URI of the signature server by placing
+arbitrarily-named YAML files into a directory on the host system,
 *_/etc/containers/registries.d_* by default. The configuration files contain a
 registry URI and a signature server URI, or _sigstore_:
 
 .Example Registries.d File
 ----
 docker:
-    registry.access.redhat.com:
-        sigstore: http://registry.access.redhat.com/content/sigstore/
+    access.redhat.com:
+        sigstore: https://access.redhat.com/webassets/docker/content/sigstore
 ----
 
-This example file may be placed on each node as
-*_/etc/containers/registries.d/redhat.com.yaml_*. It defines a signature server
-(the `sigstore` parameter) to serve signatures for the `docker` transport type
-for the Red Hat Registry (`access.registry.redhat.com`). Placing these files
-onto a cluster of nodes may be automated using Ansible, for example. No service
+In this example, the Red Hat Registry, `access.redhat.com`, is the signature
+server that provides signatures for the `docker` transport type. Its URI is
+defined in the `sigstore` parameter. You might name this file 
+*_/etc/containers/registries.d/redhat.com.yaml_*, and use Ansible to
+automatically place the file on each node in your cluster. No service
```

Comment:
Do we need the comma here: *_/etc/containers/registries.d/redhat.com.yaml_*, and

Reply:
Nope. Thanks!